apps/app-article/src/article-service.js

```diff
@@ -3,15 +3,21 @@ import {isUserLoggedIn} from "./helper";
 
 const articleApi = {
     getArticle(uuid) {
-        return fetch(config.apiUrl + "article/" + uuid, {
+        return fetch(config.production.apiUrl + "article/" + uuid, {
             method: 'GET',
             headers: attachHeaders()
         })
             .then(response => response.json())
 
     },
     getLatestArticles() {
-        return fetch(config.apiUrl + "latest?start=0&limit=5", {
+        return fetch(config.production.apiUrl + "latest?start=0&limit=5", {
```

Comment:
This should actually be `config.apiUrl`. If you look at https://github.com/KSF-Media/affresco/blob/master/apps/app-article/src/config.js, I've removed the `production` and `dev` keys there, and use `.env` to figure out which url to use (this way, it's in sync with our other deploys and workflow).

Note: the default dev `LETTERA_URL` is added here https://github.com/KSF-Media/affresco/blob/master/apps/app-article/.env